delegation-toolkit/get-started/delegation-quickstart.md

```diff
@@ -85,13 +96,14 @@ const delegateSmartAccount = await toMetaMaskSmartAccount({
 
 ### 5. Create a delegation
 
-[Create a root delegation](../how-to/create-delegation/index.md#create-a-root-delegation) from the delegator account to the delegate account.
+[Create a root delegation](../how-to/create-delegation/index.md#create-a-root-delegation) from Alice to Bob.
```

Comment:
Maybe one liner explaining root delegation, so they don't have to get redirected to other page?

Reply:
Would it be simpler to just say "create a delegation" here in the quickstart so we don't have to explain the difference between "root delegation" and "redelegation"? And the reader can click into the concepts page if they want to learn more. This is what I'll change it to, but let me know if you prefer explaining root delegation here.